### PR TITLE
creating border radius reverse

### PR DESCRIPTION
**Pull request by omariosouto**
Hi, I create this property trying make a solution for that post in circular cases: 

http://lea.verou.me/2011/03/beveled-corners-negative-border-radius-with-css3-gradients/

Live Example: https://mariosouto.com/border-radius-reverse/

### CODE REVIEW

**Comment by nucliweb** [comment]
Great work @omariosouto 

Look this @LeaVerou 🤘🏼

File: paint-worklet/border-radius-reverse/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <h2>Change values below :)</h2>
+    
+        <style style="display: block; white-space: pre;" contenteditable="true">
+        .radius-normal {
+            border-radius: 10px;
+        }
+        .radius-reverse {
+            background-color: transparent; 
+            --background-color: #ff6a00;
+            background-image: paint(border-radius-reverse);
```
Comment (surma) [inline paint-worklet/border-radius-reverse/index.html]: Do you mind giving the same technique a try with `border-image`?

**Comment by surma** [comment]
This is a great idea, @omariosouto! The only nit I have is that with your current technique of using a `background-image`, content can actually cover the corners. Ideally, you’d generate a `border-image` in the appropriate size (respecting `border-size`). Do you wanna give that a try? (Happy to help if you want!)

**Comment by omariosouto** [comment]
Hi @surma, I gave a try to border-image, and now it's working. But I guess that I find a problem when I try to zoom in or zoom out. Could you help me with that?

border-radius-reverse-zoom-error

File: paint-worklet/border-radius-reverse/border-radius-reverse.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class BorderRadiusReversePainter {
+    static get inputProperties(){
+        return ['--border-radius-reverse', '--border-radius-reverse-color']
+    }
+
+    clearCircle(context,x,y,radius) {
+        context.save();
+        context.beginPath();
+        context.arc(x, y, radius, 0, 2*Math.PI, true);
+        console.log(x , radius)
```
Comment (surma) [inline paint-worklet/border-radius-reverse/border-radius-reverse.js]: Please remove the logs before we merge this :)

File: paint-worklet/border-radius-reverse/border-radius-reverse.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        context.clip();
```
Comment (surma) [inline paint-worklet/border-radius-reverse/border-radius-reverse.js]: Smart! I like the clip approach.

**Comment by omariosouto** [comment]
Done :)

**Comment by surma** [comment]
🎉 Brilliant work. Thank you

**Pull request merged**